### ISSUE TITLE
Is there any way to implement this code on a custom dataset ?

### ISSUE THREAD

**Issue by jiteshm17** [philiptheother/FeatureDecoupling#5]
Hi, congratulations on your work!
I was actually wondering if there is any way to run this code on a custom dataset?

**Comment by philiptheother** [comment]
Hi, thanks for your question.
It is able to apply this code to a custom image dataset. In this case the PU probabilities in PU module have to be regenerated for the custom dataset if it contains many rotation-ambiguous images. The other modification is to adjust some hyper-parameters in the Non-parametric SoftMax part.
